### DISCUSSION TITLE
2021-10-27 Weekly Meeting 27 Agenda

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#111] [Weekly Meetings]
Weekly Meeting 27 is happening on Wednesday October 27, 2021 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
   - Welcome new members and attendance
   - https://github.com/omigroup/omigroup/issues/72
   - website
   - glTF subgroup
   - VKet Booth
   - WebXR Hackathon
   - https://github.com/omigroup/omigroup/issues/109
   
 ## Add Ons
  - [7 Rules of the Metaverse](https://medium.com/meta-verses/the-seven-rules-of-the-metaverse-7d4e06fa864c)

**Comment by mrmetaverse** [comment]
omi website bullets: 

-  Shadow updated [style guide](https://github.com/omigroup/website/issues/29#issuecomment-952213114)
- the website working group is going to find a new time to meet https://github.com/omigroup/website/issues/38

**Comment by mrmetaverse** [comment]
last weeks draft visions: https://github.com/omigroup/omigroup/issues/72#issuecomment-947827821

**Comment by mrmetaverse** [comment]
IdentityWoman and @jernst joined us today. Identity in the Metaverse was a recent topic brought up at an identity conference # 33 and they've come to spend time with us. Welcome!!

**Reply by jernst** [reply]
@jernst on Github.

**Comment by mrmetaverse** [comment]
Vision vote failed to advance, to revote next week after an async refinement. 
Details: 
#72 
![image](https://user-images.githubusercontent.com/63426722/139107541-695c9207-b1e2-4292-9ab6-04c87ee57a75.png)

**Comment by mrmetaverse** [comment]
website group: 
reviewed [style guide](https://github.com/omigroup/website/issues/29#issuecomment-952213114)
- Evo suggested [getting help with accessibility](https://github.com/omigroup/website/issues/39) of our style guide. 
- antpb offered to get involved by building components 
- website working group looking to https://github.com/omigroup/website/issues/38
- you can get involved in the [OMIgroup/website Repo](https://github.com/omigroup/website/projects/1).

**Comment by robertlong** [comment]
We should add an issue or discussion for tracking the process of the Vket booth / meetings

Vket2021
[Schedule] December 4 - December 18

**Reply by mrmetaverse** [reply]
https://github.com/omigroup/omigroup/issues/89 does this work?

**Comment by mrmetaverse** [comment]
We discussed the [gltf working groups recent progress](https://canary.discord.com/channels/770382203782692945/835221348443095070/902956512190824558).

**Comment by mrmetaverse** [comment]
Robert reminded the group we can partner on an upcoming webxr hackathon.

**Comment by mrmetaverse** [comment]
Mapping the Metaverse https://github.com/omigroup/omigroup/issues/109 check in

**Comment by mrmetaverse** [comment]
# OMI Core Values 

We collaborate, driven by research, privacy and accessibility, to make the metaverse more human.
We work incrementally, iteratively, and with intent.
We are creating a cooperative, sustainable culture of innovation.
We empower the people making the metaverse.
We must cultivate resiliency by championing diversity, and freedom of choice.

**Comment by mrmetaverse** [comment]
![image](https://user-images.githubusercontent.com/63426722/139109500-bbee3a04-99a2-4f52-8ca8-27df3d3ed07b.png)

**Comment by mrmetaverse** [comment]
RE: Mapping the Metaverse https://github.com/omigroup/omigroup/issues/109: 

![image](https://user-images.githubusercontent.com/63426722/139110745-ccfa1c16-38bc-4d07-8446-cd7f6b267728.png)


- https://hackmd.io/@XR/maps
- https://www.w3.org/2001/tag/doc/ethical-web-principles/

![image](https://user-images.githubusercontent.com/63426722/139112044-34b60788-7fbd-451f-b0ba-f46afd96e1e0.png)

**Comment by mrmetaverse** [comment]
(How)Do we want to respond to the [7 Rules of the Metaverse, by Tony Parisi?](https://medium.com/meta-verses/the-seven-rules-of-the-metaverse-7d4e06fa864c)
- announce our support? As they are in line with what many of us believe to be examples of what things can and should be.
